roxie/ccd/ccdserver.cpp

```diff
@@ -2741,6 +2749,16 @@ class CRoxieServerMultiInputBaseActivity : public CRoxieServerActivity
         CRoxieServerActivity::reset(); 
     }
 
+    virtual void stop()
```

Comment:
Or perhaps derive from MultiInputActivity instead. Not sure why these two base classes exist.

Reply:
(not sure if this change made any difference - it may be that it was a first attempt to fix the issue that didn't).